spec/app/domain/authentication/validate_webservice_access_spec.rb

```diff
@@ -112,7 +112,7 @@ def role_class(returned_role)
     end
 
     it "raises a NotAuthorizedInConjur error" do
```

Comment:
same here. We need to change the expectation here b/c we changed it to RoleNotAuthorizedOnA*Webservice*